Modules, Exports, and Imports.md

```diff
@@ -0,0 +1,301 @@
+# Modules, Exports, and Imports
+
+Elm's core libraries are organized into modules, as are any third-party packages you may use. For your own large
+applications, you can define your own modules. However defined, you need to import from modules to get useful values and
+types fro your program.
+
+First we'll introduce the two types of Elm programs and how they affect how you use modules. Then we'll learn how
+modules relate to files (very closely), and then how to define your own modules and control what they export. Next we'll
+learn the right and wrong ways to import from a module. Then we'll see how exports can be a form of information hiding,
+and when that is useful. Finally we'll learn a trick to organize tests or examples.
```

Comment:
I didn't like this idea at first but it turned into "After reading this guide you will know", which I like so much I might backport it to other guides.